src/LandIce/problems/LandIce_StokesFOHydrology.cpp

```diff
@@ -126,6 +126,7 @@ void LandIce::StokesFOHydrology::buildProblem (Teuchos::ArrayRCP<Teuchos::RCP<Al
 
   Intrepid2::DefaultCubatureFactory   cubFactory;
   cellCubature = cubFactory.create<PHX::Device, RealType, RealType>(*cellType, meshSpecs[0]->cubatureDegree);
+  cellCubature1pt = cubFactory.create<PHX::Device, RealType, RealType>(*cellType, 0);
```

Comment:
This is minor, but there really is no need to create a cubature in the problem, to then pass it to the evaluator. The evaluator can get the cell topo from the intrepid basis, and from that create the 1pt cubature internally. This might help keep interfaces and classes a tag smaller (avoiding confusion).

Reply:
Good point. I'll do that.